setting-up/workload-onboarding/container/index.md

```diff
@@ -70,77 +37,102 @@ docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
-        | grep -A 2 "stdout: |"
+bacalhau job describe j-6ffd54b8-e992-498f-9ee9-766ab09d5daa
 ```
 
-Expected Output:
+The output is similar to:
 
 ```shell
-stdout: |
-        Linux fff680719453 6.2.0-1019-gcp #21~22.04.1-Ubuntu SMP Thu Nov 16 18:18:34 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
-        Hello from Docker Bacalhau!
+ID            = j-6ffd54b8-e992-498f-9ee9-766ab09d5daa
+Name          = j-6ffd54b8-e992-498f-9ee9-766ab09d5daa
+Namespace     = default
+Type          = batch
+State         = Completed
+Count         = 1
+Created Time  = 2024-09-08 14:33:19
+Modified Time = 2024-09-08 14:33:20
+Version       = 0
+
+Summary
+Completed = 1
+
+Job History
+ TIME                 REV.  STATE      TOPIC       EVENT         
+ 2024-09-08 14:33:19  1     Pending    Submission  Job submitted 
+ 2024-09-08 14:33:19  2     Running                              
+ 2024-09-08 14:33:20  3     Completed                            
+
+Executions
+ ID          NODE ID     STATE      DESIRED  REV.  CREATED     MODIFIED    COMMENT      
+ e-bd5746b8  n-e002001e  Completed  Stopped  6     27m21s ago  27m21s ago  Accepted job 
+
+Execution e-bd5746b8 History
+ TIME                 REV.  STATE              TOPIC            EVENT        
+ 2024-09-08 14:33:19  1     New                                              
+ 2024-09-08 14:33:19  2     AskForBid                                        
+ 2024-09-08 14:33:19  3     AskForBidAccepted  Requesting Node  Accepted job 
+ 2024-09-08 14:33:19  4     AskForBidAccepted                                
+ 2024-09-08 14:33:19  5     BidAccepted                                      
+ 2024-09-08 14:33:20  6     Completed                                        
+
+Standard Output
+Linux 7d5c3dcc7fc2 6.5.0-1024-gcp #26~22.04.1-Ubuntu SMP Fri Jun 14 18:48:45 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
+Hello from Docker Bacalhau!
+
 ```
 
-## 4. Submit a Job With Output Files
+## 3. Submit a Job With Output Files
 
-One inconvenience that you'll see is that you'll need to mount directories into the container to access files. This is because the container is running in a separate environment from your host machine. Let's take a look at the example below:
+You always need to mount directories into the container to access files. This is because the container is running in a separate environment from your host machine.
 
-The first part of the example should look familiar, except for the Docker commands.
+The first part of this example should look familiar, except for the Docker commands.
 
 ```shell
-docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
-    docker run \
+bacalhau docker run \                                   
         --id-only \
         --wait \
         --gpu 1 \
         ghcr.io/bacalhau-project/examples/stable-diffusion-gpu:0.0.1 -- \
             python main.py --o ./outputs --p "A Docker whale and a cod having a conversation about the state of the ocean"
 ```
```

Comment:
This job was not successful for me and I kept getting the error:```
Error: not enough nodes to run job. requested: 1, available: 3, suitable: 0.
       • Node QmPLPUUj: node busy with available capacity {CPU: 0.3500000000000001, Memory: 3.7 GB, Disk: 1.7 TB, GPU: 0} and queue capacity {CPU: 1.5, Memory: 5.0 GB, Disk: 16 GB,
       GPU: 0}
       • Node QmbxGSsM: node busy with available capacity {CPU: 0.10000000000000009, Memory: 1.7 GB, Disk: 1.7 TB, GPU: 0} and queue capacity {CPU: 1, Memory: 2.0 GB, Disk: 2.8 kB,
       GPU: 0}
       • Node n-e002001e: node busy with available capacity {CPU: 0.6000000000000001, Memory: 2.7 GB, Disk: 1.7 TB, GPU: 0}```
       I am not sure if it was because of a miss at my end.